lib/App.ts

```diff
@@ -47,6 +57,8 @@ export default class App {
             const originalPath = urlParams.get("_deltaPath");
             // remove _deltaPath so we can recreate the original query string
             urlParams.delete("_deltaPath");
+            // we don't need isAuthed to be displayed to the user
+            urlParams.delete("isAuthed");
             // recreate original query string with params if necessary
             const newPath = originalPath.substring(this.basePath.length) + (urlParams.toString() ? "?" + urlParams.toString() : "");
```

Comment:
Doesn't the SPA route transformer look at every route? Could we maybe add some way in view metadata to specify a page as the root page of a SPA, then the lifecycle handler could access that instead of needing to change url parameters and redirect

Reply:
This explains a lot of what was initially confusing about the initialization of a delta route.


Comments are great